{
  try {
    super.setUp();
    fail("Should have thrown an exception");
  }
 catch (  FailedToCreateRouteException e) {
    IllegalArgumentException iae=assertIsInstanceOf(IllegalArgumentException.class,e.getCause());
    assertEquals("Constant field with name: XXX not found on Exchange.class",iae.getMessage());
  }
}
